udata/errors.py

```diff
@@ -1,4 +1,27 @@
+from flask import current_app
+
+
 class ConfigError(ValueError):
     """Exception raised when an error is detected into the configuration"""
 
     pass
+
+
+class ExceptionWithSentryDetails(Exception):
```

Comment:
I'm not sure of the usage of this Exception, since we mostly don't know when they'll occur.
Shouldn't we use [set_context](https://docs.sentry.io/platforms/python/enriching-events/context/) at the right point in time (fetching the user, etc.)

Reply:
Tags and context seem quite similar indeed, but tags might a bit better as they are searchable in the Sentry UI, it would seem: https://docs.sentry.io/platforms/javascript/enriching-events/context/

Not sure yet when this exception could be used indeed, it is mostly to align with hydra features, isn't there some events where we would want to send the associated `resource_id` when an exception occurs?